src/main/java/com/google/sps/servlets/ActionsServlet.java

```diff
@@ -28,19 +30,21 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    List<String> jsonResultList = new ArrayList<>();
+    Map<String, List<Project>> jsonResultMap = new HashMap<>();
     for (String term : terms) {
       String queryTerm = encodeTerm(term);
       Map<String, String> queryParameters = new HashMap<>();
       queryParameters.put("api_key", API_KEY);
```

Comment:
[optional] is there a more efficient way to do this so you're not putting the api key in the hashmap for every new term?

Reply:
This is true, I'll try to update it.